crate/server/src/config/params/server_params.rs

```diff
@@ -80,10 +81,32 @@ impl ServerParams {
             None
         };
 
+        let jwt_config = match (
+            &conf.auth.jwt_issuer_uri,
+            &conf.auth.jwks_uri,
+            &conf.auth.jwt_audience,
+        ) {
```

Comment:
Why not taking ownership instead?

Reply:
Because Option<Vec<IdpConfig>> does not implement the Copy trait. As discussed, I keep the reference here.

Reply:
I updated ownership as you suggested